validation/views.py

```diff
@@ -67,43 +69,34 @@ def index(request):
             all_phrases = Phrase.objects.exclude(status="auto-validated").order_by(
                 "transcription"
             )
-    elif mode == "new":
-        all_phrases = Phrase.objects.filter(status="new").order_by("transcription")
-    elif mode == "linked":
-        all_phrases = Phrase.objects.filter(status="linked").order_by("transcription")
-    elif mode == "auto-validated":
-        all_phrases = Phrase.objects.filter(status="auto-validated").order_by(
-            "transcription"
-        )
+    else:
+        all_phrases = Phrase.objects.filter(status=mode).order_by("transcription")
 
     all_phrases = all_phrases.prefetch_related("recording_set__speaker")
+    all_phrases = sorted(all_phrases, key=operator.attrgetter("transcription"))
```

Comment:
How about applying `order_by` once at the end, and letting the sort happen only on the database end?

```suggestion
    else:
        all_phrases = Phrase.objects.filter(status=mode)

    all_phrases = all_phrases.order_by('transcription').prefetch_related("recording_set__speaker")
```

Reply:
I tried to take out the `sorted` and I couldn't figure out why that changed the results. The two competing `order_by` calls would do it though--thanks!